src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -438,26 +429,23 @@ class GenesSelection extends React.Component<IGeneSelectionProps, {}> {
     @computed get hasUnsupportedOQL() {
         const geneWithUnsupportedOql = _.find(this.genesToPlot, gene => {
             if (gene.alterations && gene.alterations.length > 0) {
-                if (
-                    this.props.containerType ===
-                    AlterationContainerType.COPY_NUMBER
-                ) {
-                    let unsupportedAlteration = _.find(
-                        gene.alterations,
-                        alteration => {
+                let unsupportedAlteration = _.find(
+                    gene.alterations,
+                    alteration => {
+                        // CNAs must have '=', and 'AMP or 'HOMDEL'
+                        if (alteration.alteration_type === 'cna') {
                             return (
-                                alteration.alteration_type !== 'cna' ||
                                 alteration.constr_rel !== '=' ||
                                 !ALLOWED_CNA_TYPES.includes(
                                     alteration.constr_val!
                                 )
                             );
                         }
-                    );
-                    return unsupportedAlteration !== undefined;
-                }
-                // return true if container type in mutation
-                return true;
+                        // For mutations anything is allowed
```

Comment:
At this point in the code fusions are in fact of the mutation type. I have updated the comment to  '// For other alteration types anything is allowed'. I agree that when cBioPortal transitions to a different implementation of fusions that this logic should be revisited, but this is out of scope for this PR since here I just follow the logic that was in the code originally.

Reply:
gotcha, thanks